lib/parsers/moov_parser.rb

```diff
@@ -136,7 +136,8 @@ def parse_time_to_sample_atom(decoder, atom_tree)
 
     if stts && mdhd
       timescale = mdhd.atom_fields[:tscale]
-      sample_duration = stts.field_value(:entries).first[:sample_duration]
```

Comment:
I think you could simplify this by using `dig`: `stts.field_value(:entries).dig(0, :sample_duration)`

Reply:
I didn't know about this `dig` thanks!